### PR TITLE
docs: rework telemetry doc and add CLI warning

### PR DESCRIPTION
**Pull request by ammario**

### CODE REVIEW

**Comment by ammario** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10354** <a href="https://app.graphite.dev/github/pr/coder/coder/10354" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/coder/coder/10354?utm_source=stack-comment).

File: docs/admin/telemetry.md [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
 
-Typical SaaS companies collect far more than what we do with little transparency
-and configurability. It's hard to imagine our favorite products today existing
-without their backers having good intelligence.
+In the event we discover a critical security issue with Coder, we will use telemetry to identify affected installations and notify their administrators.
 
-We've decided the only way we can make our product open-source _and_ build at a
-fast pace is by collecting usage data as well.
+## Toggling
+You can turn telemetry on or off using either the `CODER_TELEMETRY=[true|false]` environment variable or the `--telemetry` command-line flag.
```
Comment (coadler) [inline docs/admin/telemetry.md:40]: Since you gave a `[true|false]` example for the env I think it would make sense to do it for the flag as well.

**Review by coadler — APPROVED** [review]

File: docs/admin/telemetry.md (hunk 1 above)
Comment (ammario) [reply docs/admin/telemetry.md]: Good catch

**Comment by ammario** [comment]
### Merge activity

* **Oct 19, 4:50 PM**: @ammario merged this pull request with [Graphite](https://app.graphite.dev/github/pr/coder/coder/10354).

**Pull request merged**